packages/core/src/indexing/sqlite/__tests__/read-fixture.util.ts

```diff
@@ -0,0 +1,35 @@
+import { IndexStreamArgs } from '../../database-index-api.js'
```

Comment:
`read-csv-fixture.util.ts` as the filename, to be more specific and clear what it contains?

Reply:
Ack. Also renamed `readFixture` to `readCsvFixture`.